lib/Threading/Internal/Thread.php

```diff
@@ -53,44 +53,44 @@ public function run() {
          * thread-safe objects.
          */
         $paths = [
-            \dirname(__DIR__, 5) . \DIRECTORY_SEPARATOR . 'autoload.php',
-            \dirname(__DIR__, 3) . \DIRECTORY_SEPARATOR . 'vendor' . \DIRECTORY_SEPARATOR . 'autoload.php',
+            \dirname(__DIR__, 3) . '/vendor/autoload.php',
+            \dirname(__DIR__, 5) . '/autoload.php',
```

Comment:
Why did you switch order? In almost every case (except direct usage), `\dirname(__DIR__, 5) . '/autoload.php'` will be the case matching. Given that there's a `file_exists()` below and we're not doing negative caching in PHP, we'd end up doing a stat(2) call for every run then.